extension-types.md

```diff
@@ -38,18 +38,32 @@ metadata).
 ## Extension metadata
 
 When GeoArrow-encoded Arrays have the `ARROW:extension:metadata` metadata
-field set, it must be seriaized as a UTF-8 encoded JSON object. The following
-keys in the JSON metadata object are supported:
-
-- `crs`: A JSON object describing the coordinate reference system (CRS)
-  using [PROJJSON](https://proj.org/specifications/projjson.html).
-  This key can also be omitted if the producer does not have any
-  information about the CRS. Note that regardless of the axis
-  order specified by the CRS, axis order will be interpreted
+field set, it must be seriaized as a UTF-8 encoded JSON object. The extension
+metadata specification is intentionally aligned with the
+[GeoParquet column metadata specifification](https://github.com/opengeospatial/geoparquet/blob/main/format-specs/geoparquet.md#metadata).
```

Comment:
Should this be a link to a specific git tag spec version? E.g. tag 1.1? Or do you prefer a link to `main`?